examples/gno.land/p/demo/json/README.md

```diff
@@ -0,0 +1,13 @@
+## Source
```

Comment:
What do you think about adding a section here for detailing the usage of the library?